plugin/outputStream_test.go

```diff
@@ -151,28 +144,37 @@ func (suite *OutputStreamTestSuite) TestCloseWithError() {
 	// Test that if err != nil, then the error is sent to the channel
 	stream = newOutputStream(context.Background())
 	sentErr := fmt.Errorf("an arbitrary error")
+	suite.False(stream.closed)
 	stream.CloseWithError(sentErr)
 	assertSentError(stream, sentErr)
 	suite.assertClosedChannel(stream.ch)
+	suite.True(stream.closed)
 
 	// Test that if err == ctx.Err(), then ctx.Err() is sent if a previous
 	// Write did not send it
 	ctx, cancelFunc := context.WithCancel(context.Background())
 	stream = newOutputStream(ctx)
 	cancelFunc()
+	suite.False(stream.closed)
 	stream.CloseWithError(ctx.Err())
 	assertSentError(stream, ctx.Err())
 	suite.assertClosedChannel(stream.ch)
+	suite.True(stream.closed)
 
 	// Now, test that if err == ctx.Err(), then ctx.Err() is _not_ sent if
 	// a previous Write sent it
 	stream = newOutputStream(ctx)
 	stream.sentCtxErr = true
+	suite.False(stream.closed)
 	stream.CloseWithError(ctx.Err())
 	suite.assertClosedChannel(stream.ch)
+	suite.True(stream.closed)
 
+	// TODO: Add test to ensure that consecutive closes noop
```

Comment:
I'd rather not leave these TODOs around.

Reply:
I can address them. `ExecCommand` could use some tests anyways.